blue_descriptions/urdf/gripper_v1/gripper.urdf.xacro

```diff
@@ -233,21 +248,30 @@
     </link>
     <joint
       name="${side}_r_finger_tip_joint"
-      type="continuous">
+      type="revolute">
+      <limit lower="0.5" upper="1" velocity="0.7" effort="5" />
+        <dynamics damping="0.7" friction="10.0"/>
       <origin
         xyz="0 0.06 0"
-        rpy="1.5708 -1.5754E-31 -2.5019" />
+        rpy="1.5708 -1.5754E-31 -3.5019" />
```

Comment:
Testing on my side seems to show that the parallel jaw gripper no longer acts as intended with the numerical adjustments.

Specifically referring to (on both the ${side}_r_finger_tip_joint and ${side}_l_finger_tip_joint):
* rpy change from -2.5019 to -3.019
* mimic multiplier from -1 to -0.25
* addition of the offset to be 0.85

Reply:
I'll do some testing on my side for this.

It still works on the physical right arm that I have with me though, but the visualisation doesn't work as well.

Note: I also fixed the visualisation in a separate PR here: https://github.com/berkeleyopenarms/blue_core/pull/55

That fix changes the reversal of the joint_state reporting, and has nothing to do with the offset changes.

Reply:
Ok, I've reverted the changes for the URDF description. I am working on tuning the mimic_joint_plugin parameters to try to get it to work with Gazebo.

Reply:
Alright, this should be done, feel free to take a look!